magda-ckan-connector/src/createTransformer.ts

```diff
@@ -28,7 +28,8 @@ export default function createTransformer({
             URI: URI,
             ckan: new CkanUrlBuilder({
                 name: name,
-                baseUrl: sourceUrl
+                baseUrl: sourceUrl,
+                retrievedAt: -1
```

Comment:
The `dataset-source` aspect builder is going to access this property, so if you set -1 here I'm pretty sure all the datasets will have an invalid `retrievedAt` date.